### PR TITLE
Implemented basic grid for the spreadsheet

### PR DESCRIPTION
**Pull request by Krihag**
Just basic grid made with JS, but should be flexible in using it (since we dont know design yet)

### CODE REVIEW

File: src/js/spreadsheet/cell.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import createEle from '../helpers/createEle';
+
+export default function cell(col, row) {
+  const cellContainer = createEle('div', 'relative');
+  const cellInput = createEle(
+    'input',
```
Comment (FP22FD) [inline src/js/spreadsheet/cell.js]: Hi, I was thinking if it's a good idea to have all cells as a input, it looks a pretty expensive DOM. My suggestion is to change to a DIV and I will add the input on click/focus.
But I also think that using a TABLE is more semantically correct and easier to work with. Hence each td is a cell. a tr a row, and you get for free all the role attributes.

Comment (Krihag) [reply src/js/spreadsheet/cell.js]: Good idea. Feel free to implement thoose changes

Comment (FP22FD) [reply src/js/spreadsheet/cell.js]: Don't worry, it was just a suggestion to make a more professional application, but it's not our goal now. Take your time, we have time until Friday 👍I can work around it.

Comment (Krihag) [reply src/js/spreadsheet/cell.js]: It was a good suggestion and I will add it 👍

Comment (Krihag) [reply src/js/spreadsheet/cell.js]: I added the changes. How is it now?

**Comment by Anne-Serine** [comment]
I took the liberty to adjust the html a bit and make the spreadsheet mount on an id instead of the body. 
Just run `git pull` to get the latest updates locally.

File: src/js/main.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import spreadsheet from './spreadsheet';
+import userColsAndRows from './helpers/userColsAndRows';
+
+const spreadsheetContainer = document.querySelector('#spreadsheetContainer');
+
+const [cols, rows] = userColsAndRows();
+
+spreadsheetContainer.append(spreadsheet(cols, rows));
```
Comment (itsmaik) [inline src/js/main.js:8]: main.js should be only used for importing files, all the other function and variables should be in its own file.

Comment (itsmaik) [inline src/js/main.js:8]: main.js should be only used for importing files, all the other function and variables should be in its own file.

File: src/js/helpers/ifValidNumber.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/**
+ * Checks if the given values for columns and rows are valid numbers.
+ * If the values are not valid, it throws an error.
```
Comment (itsmaik) [inline src/js/helpers/ifValidNumber.js:3]:
```
 * Checks if the given values for columns and rows are valid numbers.
 * If the values are not valid, it throws an error.
This is a test review 
```

**Pull request merged**